packages/ui/src/components/editor/properties/scene/settings/index.tsx

```diff
@@ -0,0 +1,234 @@
+/*
+CPAL-1.0 License
+
+The contents of this file are subject to the Common Public Attribution License
+Version 1.0. (the "License"); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+https://github.com/EtherealEngine/etherealengine/blob/dev/LICENSE.
+The License is based on the Mozilla Public License Version 1.1, but Sections 14
+and 15 have been added to cover use of software over a computer network and 
+provide for limited attribution for the Original Developer. In addition, 
+Exhibit A has been modified to be consistent with Exhibit B.
+
+Software distributed under the License is distributed on an "AS IS" basis,
+WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for the
+specific language governing rights and limitations under the License.
+
+The Original Code is Ethereal Engine.
+
+The Original Developer is the Initial Developer. The Initial Developer of the
+Original Code is the Ethereal Engine team.
+
+All portions of the code written by the Ethereal Engine team are Copyright © 2021-2023 
+Ethereal Engine. All Rights Reserved.
+*/
+
+import getImagePalette from 'image-palette-core'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { Color } from 'three'
+
+import { LoadingCircle } from '@etherealengine/client-core/src/components/LoadingCircle'
+import { useComponent } from '@etherealengine/ecs/src/ComponentFunctions'
+import NumericInputGroup from '@etherealengine/editor/src/components/inputs/NumericInputGroup'
+import {
+  EditorComponentType,
+  commitProperties,
+  commitProperty,
+  updateProperty
+} from '@etherealengine/editor/src/components/properties/Util'
+import { uploadProjectFiles } from '@etherealengine/editor/src/functions/assetFunctions'
+import { takeScreenshot } from '@etherealengine/editor/src/functions/takeScreenshot'
+import { generateEnvmapBake } from '@etherealengine/editor/src/functions/uploadEnvMapBake'
+import { EditorState } from '@etherealengine/editor/src/services/EditorServices'
+import {
+  blurAndScaleImageData,
+  convertImageDataToKTX2Blob,
+  imageDataToBlob
+} from '@etherealengine/engine/src/scene/classes/ImageUtils'
+import { SceneSettingsComponent } from '@etherealengine/engine/src/scene/components/SceneSettingsComponent'
+import { getState, useHookstate } from '@etherealengine/hyperflux'
+import Button from '../../../../../primitives/tailwind/Button'
+import ColorInput from '../../../../../primitives/tailwind/Color'
+import LoadingView from '../../../../../primitives/tailwind/LoadingView'
+import InputGroup from '../../../input/Group'
+import ImagePreviewInput from '../../../input/Image/Preview'
+import PropertyGroup from '../../group'
+
+export const SceneSettingsEditor: EditorComponentType = (props) => {
```

Comment:
![image](https://github.com/EtherealEngine/etherealengine/assets/3631206/b6140db8-fd48-431e-9e42-9593fdc99934)
not getting any margin on Scene setting and in general things look squished. This is at any resolution

Reply:
fixed it